packs/kubernetes/actions/db_rds_spec.py

```diff
@@ -0,0 +1,39 @@
+import re
+import uuid
+from st2actions.runners.pythonrunner import Action
+
+
+class DatabaseRdsSpec(Action):
+    def run(self, payload):
+        # For dbidentifier
+        db_name = re.sub('[^0-9a-zA-Z]+', '-', payload['name']) + "-" + payload['namespace']
```

Comment:
Assuming this would cause the action-chain to exit. That would be the intended outcome. This action-chain should not continue without it.